### ISSUE TITLE
Notice: Undefined index: items class-wc-gateway-ppec-client.php on line 602

### ISSUE THREAD

**Issue by vestaxpdx** [woocommerce/woocommerce-gateway-paypal-express-checkout#314]
Hi

If I set Subtotal Mismatch Behaviour as *Do not send line items to PayPal* when the user lands back on checkout complete page I am getting these warnings.

```
Notice: Undefined index: items in wp-content/plugins/woocommerce-gateway-paypal-express-checkout/includes/class-wc-gateway-ppec-client.php on line 602

Warning: Invalid argument supplied for foreach() in wp-content/plugins/woocommerce-gateway-paypal-express-checkout/includes/class-wc-gateway-ppec-client.php on line 602
```
I am not entirely familiar with the logic of not returning items and if that messes anything up down the line. So don't want to mess making a pull request. But it seems quite easy to dismiss this warning with a `isset($details['items'])`?

Hope we can fix soon.

**Comment by mitraval192** [comment]
@mattyza 

When you get a chance then please check it my PR and let me know if I need to do anything.

Thanks
